### ISSUE TITLE
Rebase native builds onto al2023

### ISSUE THREAD

**Issue by RichiCoder1** [maxday/lambda-perf#759]
In theory `al2023` should be a better and faster base image now than `al2`. Would love to see the native builds replaced or added complimentary to the existing `al2` builds.

**Comment by maxday** [comment]
yeah I've updated them. Do you feel I can close this issue or were you referring to something else?

**Comment by o-alexandrov** [comment]
@maxday would you prefer having:
- both al2 and al2023
- or only al2023

imho, keeping al2 isn’t giving any additional information

**Comment by maxday** [comment]
I'm closing this as I think all runtimes have now been udpated to al2023. Let me know otherwise and I can re-open
